src/main/java/com/revature/domain/QuestionEval.java

```diff
@@ -30,7 +31,7 @@
 	@JoinColumn(name="qe_qpid")
 	private QuestionPool questionPool;
 	
-	@OneToMany(mappedBy="questionEval")
+	@OneToMany(fetch = FetchType.EAGER, mappedBy="questionEval")
 	private List<QuestionComment> comments;
 	
 	public QuestionEval(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)